docs/tutorials/qiskit_pulse.rst

```diff
@@ -136,7 +141,7 @@ In the last step we perform the simulation and plot the results.
     # Start the qubit in its ground state.
     y0 = Statevector([1., 0.])
 
-    %time sol = hamiltonian_solver.solve(t_span=[0., 2*T], y0=y0, signals=signals, atol=1e-8, rtol=1e-8)
+    %time sol = hamiltonian_solver.solve(t_span=[0., 2*T], y0=y0, signals=xp, atol=1e-8, rtol=1e-8)
```

Comment:
Added a couple sentences explaining that one can either pass `xp` directly to `solve` (as we have configured the solver with pulse information), or equivalently pass `signals` as generated by the converter. 

I think for someone who is only interested in directly simulating the schedules as given, without doing any computation on the signals themselves (e.g. signal processing), this PR makes it so that there probably isn't a need to ever directly use the signal converter anymore. I agree though we should point out the different options.